lib/oli_web/live/sections/assessment_settings/settings_table_model.ex

```diff
@@ -141,6 +150,29 @@ defmodule OliWeb.Sections.AssessmentSettings.SettingsTableModel do
     """
   end
 
+  def render_available_date_column(assigns, assessment, _) do
+    assigns =
+      Map.merge(assigns, %{
+        available_date: assessment.start_date,
+        id: assessment.resource_id
+      })
+
+    ~H"""
+    <button
+      class="hover:underline whitespace-nowrap"
+      type="button"
+      phx-click={edit_date_and_show_modal(@on_edit_date, "available_date")}
+      phx-value-assessment_id={@id}
+    >
+      <%= if is_nil(@available_date) do %>
+        No available date
```

Comment:
I think we should change "No available date" to "Always available" as it better conveys to the instructor the state that an assessment is in when they have not set an available date.

Reply:
I pushed that change.

Reply:
Sounds better! Thanks